testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -169,14 +170,16 @@ public void testInsert()
     @Test
     public void testDelete()
     {
-        if (!hasBehavior(SUPPORTS_DELETE)) {
-            assertQueryFails("DELETE FROM region", "This connector does not support deletes");
-            return;
-        }
-
-        if (!hasBehavior(SUPPORTS_ROW_LEVEL_DELETE)) {
-            assertQueryFails("DELETE FROM region WHERE regionkey = 2", ".*[Dd]elet(e|ing).*(not |un)supported.*");
-            return;
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE));
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_delete", "(col bigint)", ImmutableList.of("1", "2"))) {
```

Comment:
can we just use `String tableName = "test_delete_" + randomTableSuffix();` for both "supports" and "unsupports" branches?

Reply:
Doesn't `TestTable` automatically add the suffix?

Reply:
Oh, understood. AC.